ci/run-ci.py

```diff
@@ -425,6 +425,7 @@ def run_tests():
 
     try:
         call(f'kubectl --context {KUBE_CONTEXT} exec {substra_tests_pod} -n substra-tests -- make test')
+        call(f'kubectl --context {KUBE_CONTEXT} exec {substra_tests_pod} -n substra-tests -- make test-local')
```

Comment:
For the CI we want to run the full end to end tests using the deployed backend

Reply:
With this the e2e tests run both on the deployed and mock backend, no ?

Reply:
I think `make test-local` will only run the local tests (see my comment in `tests/conftest.py`)

I would prefer the CI to only run the "remote backend" tests: if a "local backend" test fails, i don't think we want this to trigger a CI failure, which in turns triggers a message in chat.

Reply:
I would actually trigger a CI failure for tests on the local backend, as the local backend must be similar to the remote backend to be useful. Without a CI I feel that the local backend will become out-of-date quickly.

Reply:
@jmorel What do you think ?

Reply:
I agree with @Esadruhn, I think of these e2e tests as a guarantee for the user experience and not just as validation that changes in the backend / chaincode did not break anything. And in order for the local backend to be useful, we must have it as close to the remote backend as possible.